### PR TITLE
Add kube-proxy patch to allow change of base image

### PR DESCRIPTION
**Pull request by TerryHowe**
Add kube-proxy patch to 1-19

### CODE REVIEW

File: projects/kubernetes/kubernetes/1-19/patches/0012-EKS-PATCH-allow-override-of-kube-proxy-base-image [hunk 1]
```diff
@@ -0,0 +1,30 @@
+diff --git a/build/common.sh b/build/common.sh
```
Comment (vivek-koppuru) [inline projects/kubernetes/kubernetes/1-19/patches/0012-EKS-PATCH-allow-override-of-kube-proxy-base-image:1]: Are there any sign off notes we want to include here, or are we just not including them anymore?

Comment (TerryHowe) [reply projects/kubernetes/kubernetes/1-19/patches/0012-EKS-PATCH-allow-override-of-kube-proxy-base-image:11]: Notes added.

**Comment by TerryHowe** [comment]
I still need to investigate if our kube-proxy base image meets all the requirements expected, but this is good to go either way. If we need a change with the kube-proxy base, I'll make that in another PR.

**Comment by TerryHowe** [comment]
We already have logic to use AL2 for kube-proxy-base and we run `yum update` on it.

**Review by abhay-krishna — APPROVED** [review]
/lgtm
/approve

**Pull request merged**